src/plugins/auth.js

```diff
@@ -221,7 +221,10 @@ function createCredentialsProfile(credentials, payload) {
 }
 
 function extractFarmDetails(relationships) {
-  const parts = relationships.split(':')
+  const parts = relationships?.split(':')
+
+  if (!parts) return []
```

Comment:
I dont think we should return empty array. Further down we throw an error because lots of different things refer to the relationship information